src/js/hyperschedule.js

```diff
@@ -54,6 +59,7 @@ let gApiData = null;
 let gSelectedCourses = [];
 let gScheduleTabSelected = false;
 let gShowClosedCourses = true;
+let gGreyConflictCourses = greyConflictCoursesOptions[1];
```

Comment:
I don't think this can default to being enabled, because the feature you're adding introduces a considerable performance regression.

Reply:
This should be resolved in the newest commit, where the default is no graying out conflicting courses.
`let gGreyConflictCourses = greyConflictCoursesOptions[0];`